src/sonic-yang-models/doc/Configuration.md

```diff
@@ -1487,6 +1488,24 @@ name as object key and member list as attribute.
 }
 ```
 
+### Tunnel
+
+This table configures the MUX tunnel for Dual-ToR setup
+```
+{
+    "TUNNEL": {
+        "MuxTunnel0": {

```

Comment:
Can we add couple of more Tunnel entries to show all possible configs. Such as "dscp_mode": "pipe" and ecn_mode values etc.

Reply:
Sure, I'll add these and update!